Makefile

```diff
@@ -3,7 +3,7 @@ SERVER_TARGET = server
 
 CC = g++
 CFLAGS = -Wall -g -std=c++14 -I include/
-SHARED_LIBS = -lboost_serialization
+SHARED_LIBS = -lboost_serialization -lboost_thread-mt -lboost_chrono
```

Comment:
Remove lboost_chrono from LIBS if including it here. Does it not get included automatically on Mac?

Reply:
See comment below. Shouldn't need to link chrono library, can just use std::chrono.